docs/articles/documentation/getting-started/index.md

```diff
@@ -0,0 +1,195 @@
+        .click('#submit-button');
+
+    expect(await t.eval( () => document.querySelector('#article-header').text )).to.equal('Thank You, John Smith!');
+};
+```
+
+To check if the actual text is equal to the expected one, the `.equal(value)` assertion method is used.
+It is called within the special `t.eval` function provided by TestCafe. This function allows you to obtain the state of the element you verify on the client side.
+For more information, see [Executing Client Code](../test-api/executing-client-code.md).
+
+### Dialog Handling
+
+Note the Populate button on the sample page. This button fills the Developer Name text editor with a predefined name. Before it does so, a confirmation dialog is invoked. You must answer OK to proceed.
+To close this dialog from the test, you can use the special `handleConfirm` function. The following code snippet demonstrates how to do this.
+
+```js
+fixture `Getting Started - Test 3`
+    .page('http://testcafe.devexpress.com/Example');
+
+test('MyTest', async t => {
+    await t
+        .click('.populate-form')
+        .handleConfirm('OK')
+        .click('#submit-button');
+
+    expect(await t.eval( () => document.querySelector('#article-header').text )).to.equal('Thank You, Peter Parker!');
+};
+```
+
+Besides, TestCafe API provides special methods to handle other native dialogs such as alerts and prompts. For more information, see [Handling Native Dialog](../test-api/handling-native-dialogs.md).
+
+### Authentication
+
+At this step, you will see how TestCafe handles webpage authentication.
+
+The subsequent test clicks the Authentication button that performs Http Basic authentication, opens a Personal Cabinet and checks whether the user name is correct. To provide the user credentials, the `httpAuth` function is used.
+
+```js
+fixture `Getting Started - Test 4`
+    .page('http://testcafe.devexpress.com/Example')
+    .httpAuth('test', '1234');
+
+test('MyTest', async t => {
+    await t
+        .click('.populate-form')
+        .handleConfirm('OK')
+        .click('.personal-cabinet-button');
+
+    expect(await t.eval( () => document.querySelector('.user-name').text )).to.equal('Peter Parker');
+};
+```
+
+See [Http Authentication](../test-api/http-authentication.md) to learn more about authentication.
+
+## Running Test
+
+You can simply run tests from a command shell, by calling a single command where you specify target browsers and tests.
+
+```bash
+testcafe ie,chrome my-project/tests/test1.js
+```
+
+TestCafe will automatically open the chosen browsers and start test execution within them simultaneously.
+
+You can run tests in [local browsers](../using-testcafe/command-line-interface.html#local-browsers) as well as on [remote devices](../using-testcafe/command-line-interface.html#remote-browsers) (including mobile phones and tablets).
+The only requirement for a device to run TestCafe tests is that it should have network access to the TestCafe application.
```

Comment:
`TestCafe application` we've discussed it before - we don't use term `application` for the CLI version